Support 1.18

1.18 will be released 11/30.  Could you ready support for 1.18 to ease the transition?  Maybe release a build in advance?  Thank you.

ExtraHardMode produces spawn errors in 1.18:

```
[21:17:15] [Server thread/ERROR]: Could not pass event CreatureSpawnEvent to ExtraHardMode v3.15.0-6ff295d
org.bukkit.event.EventException: null
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:310) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.callCreatureSpawnEvent(CraftEventFactory.java:676) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.doEntityAddEventCalling(CraftEventFactory.java:595) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addEntity(WorldServer.java:990) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addFreshEntity(WorldServer.java:931) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.WorldAccess.lambda$0(WorldAccess.java:16) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at java.util.stream.Streams$StreamBuilderImpl.forEachRemaining(Streams.java:411) ~[?:?]
	at java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Streams.java:734) ~[?:?]
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762) ~[?:?]
	at net.minecraft.world.level.WorldAccess.addFreshEntityWithPassengers(WorldAccess.java:16) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:255) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:185) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:173) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.ChunkProviderServer.q(ChunkProviderServer.java:452) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.ChunkProviderServer.a(ChunkProviderServer.java:394) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.a(WorldServer.java:351) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1299) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:428) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1205) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1033) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:303) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NoSuchFieldError: SWAMP_HILLS
	at com.extrahardmode.features.monsters.CaveSpider.onEntitySpawn(CaveSpider.java:64) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	... 24 more
[21:17:15] [Server thread/ERROR]: Could not pass event CreatureSpawnEvent to ExtraHardMode v3.15.0-6ff295d
org.bukkit.event.EventException: null
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:310) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.callCreatureSpawnEvent(CraftEventFactory.java:676) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.doEntityAddEventCalling(CraftEventFactory.java:595) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addEntity(WorldServer.java:990) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addFreshEntity(WorldServer.java:931) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.WorldAccess.lambda$0(WorldAccess.java:16) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at java.util.stream.Streams$StreamBuilderImpl.forEachRemaining(Streams.java:411) ~[?:?]
	at java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Streams.java:734) ~[?:?]
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762) ~[?:?]
	at net.minecraft.world.level.WorldAccess.addFreshEntityWithPassengers(WorldAccess.java:16) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:255) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:185) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:173) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.ChunkProviderServer.q(ChunkProviderServer.java:452) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.ChunkProviderServer.a(ChunkProviderServer.java:394) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.a(WorldServer.java:351) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1299) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:428) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1205) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1033) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:303) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NoSuchFieldError: SWAMP_HILLS
	at com.extrahardmode.features.monsters.CaveSpider.onEntitySpawn(CaveSpider.java:64) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	... 24 more
[21:17:16] [Server thread/ERROR]: Could not pass event CreatureSpawnEvent to ExtraHardMode v3.15.0-6ff295d
org.bukkit.event.EventException: null
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:310) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.callCreatureSpawnEvent(CraftEventFactory.java:676) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.doEntityAddEventCalling(CraftEventFactory.java:595) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addEntity(WorldServer.java:990) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addFreshEntity(WorldServer.java:931) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftWorld.addEntityToWorld(CraftWorld.java:798) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.addEntity(CraftRegionAccessor.java:522) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.spawn(CraftRegionAccessor.java:498) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.spawn(CraftRegionAccessor.java:492) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.spawn(CraftRegionAccessor.java:478) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.spawnEntity(CraftRegionAccessor.java:386) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at com.extrahardmode.module.EntityHelper.spawn(EntityHelper.java:248) ~[?:?]
	at com.extrahardmode.module.EntityHelper.spawnRandomMob(EntityHelper.java:296) ~[?:?]
	at com.extrahardmode.features.monsters.MonsterRules.onEntitySpawn(MonsterRules.java:98) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.callCreatureSpawnEvent(CraftEventFactory.java:676) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.doEntityAddEventCalling(CraftEventFactory.java:595) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addEntity(WorldServer.java:990) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addFreshEntity(WorldServer.java:931) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.WorldAccess.lambda$0(WorldAccess.java:16) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at java.util.stream.Streams$StreamBuilderImpl.forEachRemaining(Streams.java:411) ~[?:?]
	at java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Streams.java:734) ~[?:?]
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762) ~[?:?]
	at net.minecraft.world.level.WorldAccess.addFreshEntityWithPassengers(WorldAccess.java:16) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:255) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:185) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:173) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.ChunkProviderServer.q(ChunkProviderServer.java:452) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.ChunkProviderServer.a(ChunkProviderServer.java:394) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.a(WorldServer.java:351) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1299) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:428) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1205) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1033) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:303) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NoSuchFieldError: SWAMP_HILLS
	at com.extrahardmode.features.monsters.CaveSpider.onEntitySpawn(CaveSpider.java:64) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	... 45 more
[21:17:16] [Server thread/ERROR]: Could not pass event CreatureSpawnEvent to ExtraHardMode v3.15.0-6ff295d
org.bukkit.event.EventException: null
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:310) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.callCreatureSpawnEvent(CraftEventFactory.java:676) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.doEntityAddEventCalling(CraftEventFactory.java:595) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addEntity(WorldServer.java:990) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addFreshEntity(WorldServer.java:931) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftWorld.addEntityToWorld(CraftWorld.java:798) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.addEntity(CraftRegionAccessor.java:522) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.spawn(CraftRegionAccessor.java:498) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.spawn(CraftRegionAccessor.java:492) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.spawn(CraftRegionAccessor.java:478) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.spawnEntity(CraftRegionAccessor.java:386) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at com.extrahardmode.module.EntityHelper.spawn(EntityHelper.java:248) ~[?:?]
	at com.extrahardmode.module.EntityHelper.spawnRandomMob(EntityHelper.java:296) ~[?:?]
	at com.extrahardmode.features.monsters.MonsterRules.onEntitySpawn(MonsterRules.java:98) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.callCreatureSpawnEvent(CraftEventFactory.java:676) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.doEntityAddEventCalling(CraftEventFactory.java:595) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addEntity(WorldServer.java:990) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addFreshEntity(WorldServer.java:931) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.WorldAccess.lambda$0(WorldAccess.java:16) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at java.util.stream.Streams$StreamBuilderImpl.forEachRemaining(Streams.java:411) ~[?:?]
	at java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Streams.java:734) ~[?:?]
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762) ~[?:?]
	at net.minecraft.world.level.WorldAccess.addFreshEntityWithPassengers(WorldAccess.java:16) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:255) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:185) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:173) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.ChunkProviderServer.q(ChunkProviderServer.java:452) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.ChunkProviderServer.a(ChunkProviderServer.java:394) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.a(WorldServer.java:351) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1299) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:428) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1205) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1033) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:303) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NoSuchFieldError: SWAMP_HILLS
	at com.extrahardmode.features.monsters.CaveSpider.onEntitySpawn(CaveSpider.java:64) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	... 45 more
[21:17:16] [Server thread/ERROR]: Could not pass event CreatureSpawnEvent to ExtraHardMode v3.15.0-6ff295d
org.bukkit.event.EventException: null
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:310) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.callCreatureSpawnEvent(CraftEventFactory.java:676) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.doEntityAddEventCalling(CraftEventFactory.java:595) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addEntity(WorldServer.java:990) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addFreshEntity(WorldServer.java:931) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.WorldAccess.lambda$0(WorldAccess.java:16) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at java.util.stream.Streams$StreamBuilderImpl.forEachRemaining(Streams.java:411) ~[?:?]
	at java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Streams.java:734) ~[?:?]
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762) ~[?:?]
	at net.minecraft.world.level.WorldAccess.addFreshEntityWithPassengers(WorldAccess.java:16) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:255) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:185) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:173) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.ChunkProviderServer.q(ChunkProviderServer.java:452) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.ChunkProviderServer.a(ChunkProviderServer.java:394) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.a(WorldServer.java:351) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1299) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:428) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1205) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1033) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:303) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NoSuchFieldError: SWAMP_HILLS
	at com.extrahardmode.features.monsters.CaveSpider.onEntitySpawn(CaveSpider.java:64) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	... 24 more
[21:17:37] [Server thread/ERROR]: Could not pass event CreatureSpawnEvent to ExtraHardMode v3.15.0-6ff295d
org.bukkit.event.EventException: null
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:310) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.callCreatureSpawnEvent(CraftEventFactory.java:676) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.doEntityAddEventCalling(CraftEventFactory.java:595) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addEntity(WorldServer.java:990) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addFreshEntity(WorldServer.java:931) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.WorldAccess.lambda$0(WorldAccess.java:16) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at java.util.stream.Streams$StreamBuilderImpl.forEachRemaining(Streams.java:411) ~[?:?]
	at java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Streams.java:734) ~[?:?]
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762) ~[?:?]
	at net.minecraft.world.level.WorldAccess.addFreshEntityWithPassengers(WorldAccess.java:16) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:255) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:185) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:173) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.ChunkProviderServer.q(ChunkProviderServer.java:452) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.ChunkProviderServer.a(ChunkProviderServer.java:394) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.a(WorldServer.java:351) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1299) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:428) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1205) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1033) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:303) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NoSuchFieldError: SWAMP_HILLS
	at com.extrahardmode.features.monsters.CaveSpider.onEntitySpawn(CaveSpider.java:64) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	... 24 more
[21:17:49] [Server thread/ERROR]: Could not pass event CreatureSpawnEvent to ExtraHardMode v3.15.0-6ff295d
org.bukkit.event.EventException: null
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:310) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.callCreatureSpawnEvent(CraftEventFactory.java:676) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.doEntityAddEventCalling(CraftEventFactory.java:595) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addEntity(WorldServer.java:990) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addFreshEntity(WorldServer.java:931) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftWorld.addEntityToWorld(CraftWorld.java:798) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.addEntity(CraftRegionAccessor.java:522) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.spawn(CraftRegionAccessor.java:498) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.spawn(CraftRegionAccessor.java:492) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.spawn(CraftRegionAccessor.java:478) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.CraftRegionAccessor.spawnEntity(CraftRegionAccessor.java:386) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at com.extrahardmode.module.EntityHelper.spawn(EntityHelper.java:248) ~[?:?]
	at com.extrahardmode.module.EntityHelper.spawnRandomMob(EntityHelper.java:296) ~[?:?]
	at com.extrahardmode.features.monsters.MonsterRules.onEntitySpawn(MonsterRules.java:98) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.callCreatureSpawnEvent(CraftEventFactory.java:676) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at org.bukkit.craftbukkit.v1_18_R1.event.CraftEventFactory.doEntityAddEventCalling(CraftEventFactory.java:595) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addEntity(WorldServer.java:990) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.addFreshEntity(WorldServer.java:931) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.WorldAccess.lambda$0(WorldAccess.java:16) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at java.util.stream.Streams$StreamBuilderImpl.forEachRemaining(Streams.java:411) ~[?:?]
	at java.util.stream.Streams$ConcatSpliterator.forEachRemaining(Streams.java:734) ~[?:?]
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762) ~[?:?]
	at net.minecraft.world.level.WorldAccess.addFreshEntityWithPassengers(WorldAccess.java:16) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:255) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:185) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.world.level.SpawnerCreature.a(SpawnerCreature.java:173) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.ChunkProviderServer.q(ChunkProviderServer.java:452) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.ChunkProviderServer.a(ChunkProviderServer.java:394) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.level.WorldServer.a(WorldServer.java:351) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1299) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:428) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1205) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1033) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:303) ~[spigot-1.18-R0.1-SNAPSHOT.jar:3335-Spigot-a5dea1c-a008701]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NoSuchFieldError: SWAMP_HILLS
	at com.extrahardmode.features.monsters.CaveSpider.onEntitySpawn(CaveSpider.java:64) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-api-1.18-R0.1-SNAPSHOT.jar:?]
	... 45 more
```

I tried compiling with the one change and got these errors in console:
`Error occurred while enabling ExtraHardMode v3.15.0-${git.commit.id.abbrev} (Is it up to date?)
Console 03:39:43
java.lang.ExceptionInInitializerError: null
at com.extrahardmode.metrics.ConfigPlotter.createPlot(ConfigPlotter.java:41) ~[ExtraHardMode-master.jar:?]
at com.extrahardmode.metrics.ConfigPlotter.<init>(ConfigPlotter.java:30) ~[ExtraHardMode-master.jar:?]
at com.extrahardmode.ExtraHardMode.onEnable(ExtraHardMode.java:212) ~[ExtraHardMode-master.jar:?]
at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[paper-api-1.18-R0.1-SNAPSHOT.jar:?]
at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[paper-api-1.18-R0.1-SNAPSHOT.jar:?]
at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[paper-api-1.18-R0.1-SNAPSHOT.jar:?]
at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugin(CraftServer.java:563) ~[paper-1.18.jar:git-Paper-36]
at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugins(CraftServer.java:477) ~[paper-1.18.jar:git-Paper-36]
at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:727) ~[paper-1.18.jar:git-Paper-36]
at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:503) ~[paper-1.18.jar:git-Paper-36]
at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:313) ~[paper-1.18.jar:git-Paper-36]
at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1202) ~[paper-1.18.jar:git-Paper-36]
at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[paper-1.18.jar:git-Paper-36]
at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.IllegalStateException: bStats Metrics class has not been relocated correctly!
at org.bstats.bukkit.Metrics.<clinit>(Metrics.java:45) ~[ExtraHardMode-master.jar:?]
... 14 more`

--- a/src/main/java/com/extrahardmode/features/monsters/CaveSpider.java
+++ b/src/main/java/com/extrahardmode/features/monsters/CaveSpider.java
@@ -61,7 +61,6 @@ public class CaveSpider extends ListenerModule
         if (entityType == EntityType.SPIDER && world.getEnvironment() == World.Environment.NORMAL
         		&& entity.getLocation().getBlock().getRelative(BlockFace.DOWN).getType() == Material.GRASS
                 && entity.getLocation().getBlock().getBiome() == Biome.SWAMP
-                || entity.getLocation().getBlock().getBiome() == Biome.SWAMP_HILLS
                 && event.getSpawnReason() == CreatureSpawnEvent.SpawnReason.NATURAL)
         {
             if (plugin.random(cavespiderSpawnPercent))

